Vagrantfile

```diff
@@ -247,6 +247,7 @@ EOF
         if [[ $selinux == Enforcing ]]; then
             setenforce 0
         fi
+        modprobe overlay
```

Comment:
Shouldn't we use `sudo systemctl restart systemd-modules-load.service`?

Reply:
In the end the `systemd-modules-load.service` does nothing else than a `modprobe`.  The only difference is that it collects the information about which modules to "modprobe" from the files `/etc/modules-load.d/*.conf`, `/run/modules-load.d/*.conf`, `/usr/local/lib/modules-load.d/*.conf` and `/usr/lib/modules-load.d/*.conf`.